client/src/authorizationPage/authorizePage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import st from './authorizePageStyle.module.css'
```

Comment:
```suggestion
import st from './authorize_page_style.module.css'
```

Го называть модули дип кейсом

Reply:
ток модули или все файлы?

Reply:
Только модули